### ISSUE TITLE
GHA notifications say tagged/refs/heads/master, could they say just tagged/master?

### ISSUE THREAD

**Issue by phadej** [haskell-CI/haskell-ci#452]

**Comment by RyanGlScott** [comment]
This is displayed like it is currently due to the IRC template's use of [`github.ref`](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context). This is a bit verbose, but it also indicates whether the workflow is running on a branch (e.g., `tagged/refs/heads/master`) or in a PR (e.g., `distributive/refs/pull/59/merge`).

I'm sure there's a way to make this prettier, although I'll admit that I haven't spent much time investigating the various options at https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context.

**Comment by phadej** [comment]
I realized that it is actually the full length git commit hash which makes message overly long. For our purposes 10 first characters would be enough. 

However, it would be a hack https://github.community/t/substring-function/18401/2

**Comment by RyanGlScott** [comment]
Yeah, I had a feeling that we might need to resort to `cut`-like hackery at some point. I tried to delay the inevitable as much as possible, but here we are :)

**Comment by phadej** [comment]
![Screenshot from 2020-12-30 02-45-10](https://user-images.githubusercontent.com/51087/103322947-1af0f880-4a49-11eb-8c36-8075bc4228e7.png)

This misalignment makes me anxious :S

**Comment by phadej** [comment]
Another option is to write own bot and just `curl` the `github` JSON on it.

Then we'd have much more control over notifications (and sane language to handle them).

(Not that I particularly keen in maintaining a bot, but maybe it will be easier on Freenode servers as well...)
